upgrade-instructions.md

```diff
@@ -69,6 +69,28 @@ we regenerate invalid UidValidity upon reads.
 While this sanitizing is transparent to the end user and the admin, it might lead to rare IMAP client full mailbox
  resynchronisation. (one chance out of two billions).
 
+### Tune Cassandra time serie tables options
+
+Date 18/03/2020
+
+SHA-1 a6613bc91b c8792c64bf
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3121
+
+Our usage of Cassandra for the time series have been improved by fine tuning the compaction strategy and
+the read repair option.
+
+#### Upgrade procedure
+
+Alter `deletedMailsV2` and `enqueuedMailsV3` with this `cqlsh` commands:
```

Comment:
In order to unlock these improvements we advise you running the following commands on existing instalations